site/guide/0-introduction.md

```diff
@@ -1,10 +1,10 @@
 # Introduction
 
 Rocket is a web framework for Rust. If you'd like, you can think of Rocket as
-being a more flexible, friendly medley of [Rails](http://rubyonrails.org),
-[Flask](http://flask.pocoo.org/),
-[Bottle](http://bottlepy.org/docs/dev/index.html), and
-[Yesod](http://www.yesodweb.com/). We prefer to think of Rocket as something
+being a more flexible, friendly medley of [Rails](https://rubyonrails.org/),
+[Flask](https://flask.palletsprojects.com/),
```

Comment:
Note: I used this URL as it's what http://flask.pocoo.org/ redirects to but the main Flask page is https://palletsprojects.com/p/flask/ instead. I wasn't 100% sure which of those would be preferable to use.

Reply:
I would be in favor of the main page, to match rails and yesod (bottle doesn't seem to have a separate home page).